### ISSUE TITLE
[PLATFORM UI] Backup failures/issues in 2.5.0.0 - B1 build

### ISSUE THREAD

**Issue by nmantrav2** [yugabyte/yugabyte-db#6276]
1. YSQL universe backup on puppy-food-4s failed without any error details - http://portal.dev.yugabyte.com/universes/38074303-ccc2-4a7e-970e-ae6fafc60c3d/backups (taken at 9:18 AM PST with 2.5 # B1). YSQL universe backup completed OK on another universe - http://portal.dev.yugabyte.com/universes/81fd5433-56de-4d09-9887-b29e6593d503/backups. Not sure why puppy-food-4s one failed.

2. YCQL table with 1.26 GB data backup on puppy-food-4s seems to be spinning. Started at 9:19 AM PST and didn't finish by 10:20 AM PST - http://portal.dev.yugabyte.com/universes/38074303-ccc2-4a7e-970e-ae6fafc60c3d/backups

**Comment by daniel-yb** [comment]
1. Failed due to duplicate table name: `Namespace 'ysql.postgres' has multiple tables named 'sqlsecondaryindex_index', specify table id instead`
2. Failed due to being started before (1) completed: `Error executing task BackupUniverse(38074303-ccc2-4a7e-970e-ae6fafc60c3d) with error='A backup for this universe is already in progress.'.`
we do not allow multiple concurrent backup tasks on the same universe

**Comment by streddy-yb** [comment]
@daniel-yb - do you recall what the resolution for this issue was?

**Comment by daniel-yb** [comment]
@streddy-yb I'm not sure I understand which issue you're referring to exactly.
For (1), I tagged @OlegLoginov on the Slack thread regarding this issue as it looks to be out of scope of a YW specific issue. Not sure if he's had time to follow up on it yet or not.
For (2) this was resolved with #5888. If you're asking how to move past this issue in an existing version of YW from before this fix went in, it requires manually updating some DB entries for the universe, backup, task_info, and customer_task table.
